src/cil.ml

```diff
@@ -7000,11 +7025,6 @@ let initCIL () =
         Some machine -> M.theMachine := machine
       | None -> M.theMachine := if !msvcMode then M.msvc else M.gcc
     end;
-    (* Pick type for string literals *)
-    stringLiteralType := if !M.theMachine.M.const_string_literals then
-      charConstPtrType
-    else
-      charPtrType;
```

Comment:
Is removing this necessary?

Reply:
Ok, looks like this is an MSVC thing, which we will no longer support anyway.

Reply:
I think it was actually supposed to be a GCC thing, but it didn't even match the typing behavior of GCC: https://github.com/goblint/cil/pull/48#issuecomment-918283984.